x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -102,7 +102,13 @@ primaryExpression
     ;
 
 functionExpression
-    : identifier LP (ASTERISK | (booleanExpression (COMMA booleanExpression)*))? RP
+    : functionName LP (ASTERISK | (booleanExpression (COMMA booleanExpression)*))? RP
+    ;
+
+functionName
+    // Additional function identifiers that are already a reserved word in the language
+    : {this.isDevVersion()}? DEV_MATCH
```

Comment:
The `Match` command is gone, it is a good time to remove the `DEV_MATCH` command from lexer and consolidate it with the `DEV_MATCH_OP`.

Reply:
Makes sense, done in 19f05465e146f831807aa0f714074ac5192fce6e